src/lib/components/Logo.svelte

```diff
@@ -0,0 +1,112 @@
+<script lang="ts">
+	export const { hex = '#303326' } = $props();
```

Comment:
nit: can we replace this with `var(--charcoal)`?

Reply:
I'm pretty sure that will only work within CSS. But I think I can finagle that to work here